devtools/release_build.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+#
+# Build release archives for Linux, Darwin and Windows.
+#
+
+# Compute package base name
+if [[ -z "${NAME}" ]]; then
+  echo "ERROR: Unable to determine package name"
+  exit 1
+fi
+
+if [[ ! -z "${VERSION}" ]]; then
+  SUFFIX=${VERSION}
+else
+  COMMIT="$(git rev-parse --short HEAD)"
+  SUFFIX="$(date +%Y%m%d%H)-${COMMIT}"
+fi
+
+if [[ -z "${SUFFIX}" ]]; then
+  echo "ERROR: Unable to determine version"
+  exit 1
+fi
+
+BASE_FILENAME="${NAME}-${SUFFIX}"
+
+
+# Build program for multiple platforms
+
+export GOOS=linux
+export GOARCH=386
+FILENAME="${BASE_FILENAME}.${GOOS}.${GOARCH}.tar.gz"
+echo "Building ${FILENAME}"
+go build
+tar -cvzf ${FILENAME} ${NAME}
+rm ${NAME}
```

Comment:
do I understand this right that the build step is repeated exactly the same for all variants and just the archive filename differs? if so why not just build once and copy the archive to different names?

Reply:
It is true that the same step is repeated four times, but it uses different values for the environment variables `GOOS` and `GOARCH` each time.